app/src/main/java/com/dessalines/thumbkey/keyboards/PLMessagEase.kt

```diff
@@ -0,0 +1,1268 @@
+                                KeyC(
+                                    display = KeyDisplay.TextDisplay("\""),
+                                    action = KeyAction.CommitText("\""),
+                                    color = ColorVariant.MUTED,
+                                ),
+                        ),
+                ),
+                KeyItemC(
+                    center =
+                        KeyC(
+                            display = KeyDisplay.TextDisplay("9"),
+                            action = KeyAction.CommitText("9"),
+                            size = FontSizeVariant.LARGE,
+                            color = ColorVariant.PRIMARY,
+                        ),
+                    swipeType = SwipeNWay.FOUR_WAY_DIAGONAL,
+                    swipes =
+                        mapOf(
+                            SwipeDirection.TOP to
+                                KeyC(
+                                    display = KeyDisplay.TextDisplay("&"),
+                                    action = KeyAction.CommitText("&"),
+                                    color = ColorVariant.MUTED,
+                                ),
+                            SwipeDirection.TOP_RIGHT to
+                                KeyC(
+                                    display = KeyDisplay.TextDisplay("°"),
+                                    action = KeyAction.CommitText("°"),
+                                ),
+                            SwipeDirection.RIGHT to
+                                KeyC(
+                                    display = KeyDisplay.TextDisplay(">"),
+                                    action = KeyAction.CommitText(">"),
+                                    color = ColorVariant.MUTED,
+                                ),
+                            SwipeDirection.BOTTOM_LEFT to
+                                KeyC(
+                                    display = KeyDisplay.TextDisplay(";"),
+                                    action = KeyAction.CommitText(";"),
+                                    color = ColorVariant.MUTED,
+                                ),
+                            SwipeDirection.LEFT to
+                                KeyC(
+                                    display = KeyDisplay.TextDisplay("#"),
+                                    action = KeyAction.CommitText("#"),
+                                ),
+                        ),
+                ),
+                BACKSPACE_KEY_ITEM,
+            ),
+            listOf(
+                KeyItemC(
+                    center =
+                        KeyC(
+                            display = KeyDisplay.TextDisplay("0"),
+                            action = KeyAction.CommitText("0"),
+                            size = FontSizeVariant.LARGE,
+                            color = ColorVariant.PRIMARY,
+                        ),
+                ),
+                SPACEBAR_KEY_ITEM,
+                RETURN_KEY_ITEM,
```

Comment:
It brakes spacing with 0 as in my comment


Here is proposed fix:
```diff
diff --git a/app/src/main/java/com/dessalines/thumbkey/keyboards/PLMessagEase.kt b/app/src/main/java/com/dessalines/thumbkey/keyboards/PLMessagEase.kt
index 1ace514..8f64d17 100644
--- a/app/src/main/java/com/dessalines/thumbkey/keyboards/PLMessagEase.kt
+++ b/app/src/main/java/com/dessalines/thumbkey/keyboards/PLMessagEase.kt
@@ -1249,8 +1249,9 @@ val KB_PL_MESSAGEASE_NUMERIC =
                             size = FontSizeVariant.LARGE,
                             color = ColorVariant.PRIMARY,
                         ),
+                    widthMultiplier = 3
                 ),
-                SPACEBAR_KEY_ITEM,
+                SPACEBAR_SKINNY_KEY_ITEM,
                 RETURN_KEY_ITEM,
             ),
         ),
```

Reply:
This just got merged, so fix this in a PR if you would.

Reply:
ok, I will create one this week 👍

Reply:
Actually I'm gonna do a release in a min, so I'll correct this now.

Reply:
Unfortunately I've found some more small mistakes 😞 

- the minus swipe doesnt produce minus
- it lacks forwardtick
- some special characters have too important color

Reply:
You'll have to fix those in later PRs.

Reply:
Addressed in https://github.com/dessalines/thumb-key/pull/777